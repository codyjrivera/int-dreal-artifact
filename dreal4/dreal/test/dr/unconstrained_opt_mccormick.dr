# McCormick
# https://en.wikipedia.org/wiki/Test_functions_for_optimization#Test_functions_for_single-objective_optimization

var:
    [-1.5, 4] x;
    [-3, 4] y;

cost:
    sin(x + y) + (x - y)^2 - 1.5 * x + 2.5 * y + 1;
