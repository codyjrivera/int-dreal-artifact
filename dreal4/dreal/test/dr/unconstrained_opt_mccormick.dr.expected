delta-sat with delta = 0.2
